.jenkins/infrastructure/build_azure_managed_images.Jenkinsfile

```diff
@@ -148,7 +149,7 @@ def buildWindowsManagedImage(String os_series, String img_name_suffix, String la
                                 --resource-group ${VM_RG_NAME} \
                                 --location ${REGION} \
                                 --name ${VM_NAME} \
-                                --size Standard_DC4s_v2 \
+                                --size Standard_DC4s_v3 \
```

Comment:
Note: Leaving this here for future reference. I found that scripts/windows-install-prerequisites.ps1 failed sporadically for DC4s_v3. This was caused by not being able to find AESMService.

Reply:
Thanks for the heads up Rathna!